crates/pathfinder/src/storage/state.rs

```diff
@@ -1274,38 +1289,26 @@ mod tests {
 
                 #[test]
                 fn some() {
-                    let storage = Storage::in_memory().unwrap();
-                    let connection = storage.connection().unwrap();
-
-                    let blocks = create_blocks();
-                    for block in &blocks {
-                        StarknetBlocksTable::insert(&connection, block).unwrap();
-                    }
-
-                    for block in blocks {
-                        let result = StarknetBlocksTable::get(&connection, block.number.into())
-                            .unwrap()
-                            .unwrap();
-
-                        assert_eq!(result, block);
-                    }
+                    with_default_blocks(|tx, blocks| {
+                        for block in blocks {
+                            let result = StarknetBlocksTable::get(tx, block.number.into())
+                                .unwrap()
+                                .unwrap();
+
+                            assert_eq!(result, block);
+                        }
+                    })
```

Comment:
These were the only ones I found the energy to do, but probably something similar is needed elsewhere.